google/cloud/pubsub/doc/pubsub-main.dox

```diff
@@ -102,31 +102,31 @@ Follow these links to find examples for other `*Client` classes:
 
 /*! @page Publisher-endpoint-snippet Override @c Publisher Default Endpoint
 
-@snippet google/cloud/pubsub/samples/client_samples.cc publisher-set-endpoint
+@snippetgoogle/cloud/pubsub/admin/samples/topic_admin_client_samples.cc publisher-set-endpoint
```

Comment:
I imagine the `publish-docs` build will fail. We can only move the generate admin samples. (The ones with `TopicAdminClient`, `SubscriptionAdminClient`)

Reply:
I see now. Done.